build/charts/antrea-windows/conf/ovs/Run-AntreaOVS.ps1

```diff
@@ -22,18 +22,21 @@ ovs-vsctl --no-wait init
 $OVS_VERSION=$(Get-Item $OVSDriverDir\OVSExt.sys).VersionInfo.ProductVersion
 ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
 
-ovs-vswitchd --log-file=/var/log/antrea/openvswitch/ovs-vswitchd.log --pidfile -vfile:info --detach
-
+$RetryInterval = 2
```

Comment:
Shall we add some comments here to explain why we introduce another interval?

Reply:
Sure, updated, PTAL.